src/main/resources/view/StrategyPanel.fxml

```diff
@@ -1,15 +1,17 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import javafx.scene.control.Slider?>
 <?import javafx.scene.image.ImageView?>
 <?import javafx.scene.layout.AnchorPane?>
 <?import javafx.scene.layout.Pane?>
 
-
 <AnchorPane fx:id="strategyAnchorPane" maxHeight="-Infinity" maxWidth="-Infinity"
             minHeight="-Infinity" minWidth="-Infinity"
             AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0"
             AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0"
             xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
    <Pane fx:id="playerView"/>
-   <ImageView fx:id="strategyImage" fitHeight="150.0" fitWidth="200.0" pickOnBounds="true" preserveRatio="true" />
+   <ImageView fx:id="strategyImage" fitHeight="161.0" fitWidth="204.0" layoutX="25.0" layoutY="27.0" pickOnBounds="true" preserveRatio="true" />
+   <Slider layoutY="24.0" orientation="VERTICAL" prefHeight="161.0" prefWidth="25.0" showTickLabels="true" showTickMarks="true" styleClass="axis-left" stylesheets="@Extensions.css" value="50.0" AnchorPane.bottomAnchor="0.0" AnchorPane.leftAnchor="0.0" AnchorPane.topAnchor="24.0" />
+   <Slider layoutX="26.0" max="200.0" prefHeight="25.0" prefWidth="204.0" showTickLabels="true" showTickMarks="true" styleClass="axis-top" stylesheets="@Extensions.css" AnchorPane.leftAnchor="26.0" AnchorPane.rightAnchor="0.0" AnchorPane.topAnchor="0.0" />
```

Comment:
There appears to be an issue with the slider in that player icons appear behind the slider:

![image](https://user-images.githubusercontent.com/77106335/160524550-c115f153-cebe-4344-9427-2a24ef4930a9.png)


To fix this, Either the slider should be included in the `StrategyPanel `class and initialized with the following method:

`sliderID.toBack()`

or include the following method in the initialization of the circle objects (representing players) as such:

`circle.toFront()`

Reply:
Also, the slider appears to scale along the window. This could cause issues with the `move player` function (depending on how it will be implemented).

Reply:
As for how to invert the y-axis, try to utilize this implementation: 

https://stackoverflow.com/questions/53879734/how-to-flip-axis-of-javafx-slider

Reply:
Thanks for the suggestion Abdul. I actually referred to this post when implementing the slider. However, the proposed solution here does not seem to work